examples/user_data.cpp

```diff
@@ -33,7 +34,7 @@ int main(int argc, char* argv[])
     // sleep for a bit so this thread doesn't grab the active request count too fast and just shutdown
     std::this_thread::sleep_for(500ms);
 
-    while (event_loop.GetActiveRequestCount() > 0) {
+    while (event_loop.HasUnfinishedRequests()) {
```

Comment:
Would `HasActiveRequests` be a better name?

Reply:
I went with HasUnfinishedRequests because it checks if we have active OR pending requests. I wouldn't mind HasActiveRequests if it is less confusing though.

Reply:
Your call